common/consts.go

```diff
@@ -3,30 +3,36 @@ package common
 
 import "github.com/cockroachdb/apd/v3"
 
+// Daemon default ports and URLs
```

Comment:
Apparently, we can group the `const` blocks around related values and then just give the group one comment and the linter is happy. It has to be multiple blocks, because if the comment is inside the `const` block, it is associated only with one value.